examples/02_decoding/plot_haxby_anova_svm.py

```diff
@@ -21,8 +21,13 @@
 #############################################################################
 # Load the behavioral data
 import numpy as np
-y, session = np.loadtxt(haxby_dataset.session_target[0]).astype("int").T
-conditions = np.recfromtxt(haxby_dataset.conditions_target[0])['f0']
+labels = np.recfromcsv(haxby_dataset.session_target[0], delimiter=" ")
+conditions = labels['labels']
+categories = np.unique(conditions)
+y = np.zeros_like(conditions)
+for c, category in enumerate(categories):
+    y[conditions == category] = c
```

Comment:
Are you sure that "y" needs to be numerical? I think that you could take "conditions" and use it as the "y" vector, which would simplify the example.